src/sidebar/store/modules/frames.js

```diff
@@ -40,20 +44,27 @@ const actions = util.actionTypes(update);
 
 /**
  * Add a frame to the list of frames currently connected to the sidebar app.
+ *
+ * @param {Frame} frame
  */
 function connectFrame(frame) {
   return { type: actions.CONNECT_FRAME, frame: frame };
 }
 
 /**
  * Remove a frame from the list of frames currently connected to the sidebar app.
+ *
+ * @param {Frame} frame
  */
 function destroyFrame(frame) {
   return { type: actions.DESTROY_FRAME, frame: frame };
 }
 
 /**
  * Update the `isAnnotationFetchComplete` flag of the frame.
+ *
+ * @param {string} uri
+ * @param {boolean} status
```

Comment:
This is a pre-existing issue, but `status` is not a great name for a boolean variable. `isFetchComplete` would be better.